lib/ansible/modules/notification/cisco_spark.py

```diff
@@ -62,53 +62,53 @@
 # Note: The following examples assume a variable file has been imported
 # that contains the appropriate information.
 
-- name: Cisco Spark - Markdown Message to a Room
+- name: Cisco Webex Teams - Markdown Message to a Room
   cisco_spark:
     recipient_type: roomId
     recipient_id: "{{ room_id }}"
     message_type: markdown
     personal_token: "{{ token }}"
-    message: "**Cisco Spark Ansible Module - Room Message in Markdown**"
+    send_message: "**Cisco Webex Teams Ansible Module - Room Message in Markdown**"
 
-- name: Cisco Spark - Text Message to a Room
+- name: Cisco Webex Teams - Text Message to a Room
   cisco_spark:
     recipient_type: roomId
     recipient_id: "{{ room_id }}"
     message_type: text
     personal_token: "{{ token }}"
-    message: "Cisco Spark Ansible Module - Room Message in Text"
+    send_message: "Cisco Webex Teams Ansible Module - Room Message in Text"
 
-- name: Cisco Spark - Text Message by an Individuals ID
+- name: Cisco Webex Teams - Text Message by an Individuals ID
   cisco_spark:
     recipient_type: toPersonId
     recipient_id: "{{ person_id}}"
     message_type: text
     personal_token: "{{ token }}"
-    message: "Cisco Spark Ansible Module - Text Message to Individual by ID"
+    send_message: "Cisco Webex Teams Ansible Module - Text Message to Individual by ID"
 
 - name: Cisco Spark - Text Message by an Individuals E-Mail Address
   cisco_spark:
     recipient_type: toPersonEmail
     recipient_id: "{{ person_email }}"
     message_type: text
     personal_token: "{{ token }}"
-    message: "Cisco Spark Ansible Module - Text Message to Individual by E-Mail"
+    send_message: "Cisco Webex Teams Ansible Module - Text Message to Individual by E-Mail"
 
 """
 
 RETURN = """
 status_code:
   description:
-    - The Response Code returned by the Spark API.
-    - Full Response Code explanations can be found at U(https://developer.ciscospark.com/endpoint-messages-post.html).
+    - The Response Code returned by the Webex Teams API.
+    - Full Response Code explanations can be found at U(https://https://developer.webex.com/docs/api/v1/messages/create-a-message).
   returned: always
   type: int
   sample: 200
 
-message:
+send_message:
     description:
-      - The Response Message returned by the Spark API.
-      - Full Response Code explanations can be found at U(https://developer.ciscospark.com/endpoint-messages-post.html).
+      - The Response Message returned by the Webex teams API.
+      - Full Response Code explanations can be found at U(https://developer.webex.com/docs/api/v1/messages/create-a-message).
```

Comment:
```suggestion
      - Full Response Code explanations can be found at U(https://developer.webex.com/docs/api/v1/messages/create-a-message).
      - C(message) alias is deprecated in Ansible 2.10 and will be removed in Ansible 2.14, since it is used internally by Ansible Core Engine. 
    aliases: [ 'message' ]
```